src/main/java/rife/bld/operations/PublishOperation.java

```diff
@@ -348,12 +348,12 @@ protected String executeSignFile(File file)
         try {
             process.waitFor();
         } catch (InterruptedException e) {
-            throw new SignException(file, e.getMessage());
+            throw new SignException(file, e);
```

Comment:
This was on purpose to not get a full stack trace reported as the result of this command